packages/sources/bank-frick-test/src/config/index.ts

```diff
@@ -9,6 +9,17 @@ export const signingAlgorithms: SigningAlgorithm[] = ['rsa-sha256', 'rsa-sha384'
 const MAX_PAGE_SIZE = 500
 
 export const customSettings = {
+  API_ENDPOINT: {
+    description: 'The endpoint to use for making requests to the Bank Frick API',
+    type: 'string',
+    required: false,
+    default: 'https://olbsandbox.bankfrick.li',
```

Comment:
Noticed a discrepancy in this default from the old env var override `https://olbsandbox.bankfrick.li/webapi/v2`. Is this intentional? Don't think I see `/webapi/v2` appended later.